targets/frontend/src/lib/idGenerators.js

```diff
@@ -0,0 +1,23 @@
+import { v4 as uuidv4 } from "uuid";
```

Comment:
I think it could be a good idea to move it as a dedicatad package in shared workspace

Reply:
agree